static/js/ui-utils.js

```diff
@@ -187,19 +187,6 @@ function toggleDrawer(id, openCallback, closeCallback) {
 
 }
 
-function initTreatmentDrawer()  {
-  $('#eventType').val('BG Check');
-  $('#glucoseValue').val('').attr('placeholder', 'Value in ' + browserSettings.units);
-  $('#meter').prop('checked', true);
-  $('#carbsGiven').val('');
-  $('#insulinGiven').val('');
-  $('#preBolus').val(0);
-  $('#notes').val('');
-  $('#enteredBy').val(browserStorage.get('enteredBy') || '');
-  $('#nowtime').prop('checked', true);
-  $('#eventTimeValue').val(currentTime());
-}
-
 function currentTime() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['currentTime' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/784416117/issues/source?bid=2452379&fileBranchId=2490906#l190)